charts/hono/README.md

```diff
@@ -388,78 +495,14 @@ The Helm chart supports deployment of an example data grid which can be used for
-```
-
-#### Data Grid based Implementation
-
-Hono also contains a production ready, data grid based implementation of the Device Connection API which can be deployed
-and used instead of the example implementation. The service component can be deployed by means of setting the
-*deviceConnectionService.enabled* property to `true`.
-
-The service requires a connection to a data grid for storing the device connection data.
-The Helm chart supports deployment of an example data grid which can be used for experimenting by means of setting the
-*dataGridExample.enabled* property to `true`:
-
-```bash
-helm install --dependency-update --wait -n hono --set useCommandRouter=false --set deviceConnectionService.enabled=true --set dataGridExample.enabled=true eclipse-hono eclipse-iot/hono 
-```
-
-This will deploy the data grid based Device Connection service and configure all protocol adapters to use it instead of
-the example Device Registry implementation.
-
-The Device Connection service can also be configured to connect to an already existing data grid. Use the *dataGridSpec*
-property for this.
-
-Setting the *deviceConnectionService.enabled* property to `true` and neither setting *dataGridExample.enabled* to `true`
-nor configuring an already existing data grid using the *dataGridSpec* property will result in the Device Connection
-service using an embedded cache for storage. This is a lightweight deployment option but not suitable for production
-purposes.
-
-## Enabling or disabling Protocol Adapters
-
-The Helm chart by default installs the HTTP, MQTT and AMQP protocol adapters.
-However, the chart also supports deployment of additional protocol adapters which are still considered experimental or
-have been deprecated.
-The following table provides an overview of the corresponding configuration properties that need to be set during installation.
-
-| Property                     | Default  | Description                              |
-| :--------------------------- | :------- | :--------------------------------------- |
-| *adapters.amqp.enabled*      | `true`    | Indicates if the AMQP protocol adapter should be deployed. |
-| *adapters.coap.enabled*      | `false`   | Indicates if the CoAP protocol adapter should be deployed. |
-| *adapters.http.enabled*      | `true`    | Indicates if the HTTP protocol adapter should be deployed. |
-| *adapters.kura.enabled*      | `false`   | Indicates if the deprecated Kura protocol adapter should be deployed. |
-| *adapters.lora.enabled*      | `false`   | Indicates if the (experimental) LoRa WAN protocol adapter should be deployed. |
-| *adapters.mqtt.enabled*      | `true`    | Indicates if the MQTT protocol adapter should be deployed. |
-
-The following command will deploy the LoRa adapter along with Hono's standard adapters (AMQP, HTTP and MQTT):
-
-```bash
-helm install --dependency-update --wait -n hono --set adapters.lora.enabled=true eclipse-hono eclipse-iot/hono
-```
-
 ## Jaeger Tracing
```

Comment:
I think everything below the `kubectl get service` command in this chapter can be removed.
Specifically, the `jaegerAgentConf` property and `SAMPLING_STRATEGIES_FILE` environment variable aren't supported any more.

The `example/jaeger/default-sampling-strategies.json` file is obsolete as well.